spec/v2/index.bs

```diff
@@ -1699,6 +1626,160 @@ For each new ProductFootprint, the [=data owner=] MUST
+
+The following `$id-type$` values are recommended.
+
+<figure id="pdf-product-id-urn-entities">
+<table class="data">
+<thead>
+  <tr>
+    <td>`$id-type$`
+    <td>Description
+    <td>Notes
+<tbody>
+  <tr>
+    <td>`product-id`
+    <td>Specifies a product id from a third-party organization, standard, etc. 
+    <td>Use this entity-idtype when no other, more specific one is applicable.
+  <tr>
+    <td>`buyer-id`
+    <td>Specifies a product id created by the buyer, aka “data recipient”
+    <td>This is the equivalent of "buyer-assigned" as referenced in Tech Specs V2.
+  <tr>
+    <td>`supplier-id`
+    <td> Specifies a product id created by the supplier, aka “data owner”
+    <td>This is the equivalent of "vendor-assigned" as referenced in Tech Specs V2
+</table>
+</figure>
+
+This is a non-exhaustive list of `$entity-idtype$`. This set MAY be extended in minor versions of the standard release. Organizations may contact PACT to propose additional `$entity-idtype$` for consideration to be added as recommended industry-agnostic identifiers. 
+
+Organizations (especially organizations initiatives) are encouraged to define the relevant `$entity-idtype$` for products within their industry separately. 
+
+## Examples ## {#product-identifier-examples}
+
+Below is a list of examples of <{ProductFootprint/productIds}> as used in the <{ProductFootprint}> datat type clarifying the use of well-known and custom URN namespaces for identifying products. 
+
+<figure id="product-id-examples">
+<table class="data">
+<thead>
+  <tr>
+    <td>Product ID type
+    <td>Example
+<tbody>
+  <tr>
+    <td>
+    Company-specific
+    
+    Identifiers using the `pact` namespace, created by a given company for the purposes of uniquely identifying their products
+
+    <td><div class=example>
+    ```json
+    ["urn:pact:sample.com:product-id:44055-9c05bc35-68f8"]
+    ```</div>
+    <div class=example>
+    ```json
+    ["urn:pact:sample-buyer.com:buyer-id:103403453"]
+    ```</div>
+    <div class=example>
+    ```json
+    ["urn:pact:sample-supplier.com:supplier-id:1234"]
+    ```</div>
+  <tr>
+    <td>ISBN
+
+    Well known ISBN standard, see [iana.org](https://www.iana.org/assignments/urn-formal/isbn)
+    <td><div class=example>
+    ```json
+    ["urn:isbn:978-951-0-18435-6"]
+    ```
+  <tr>
+    <td>GTIN (widely used)
+
+    GTIN is not an official IANA registered namespace, however in practice it is used to specify GTINs as a URN. See [gs1.org](https://www.gs1.org/standards/id-keys/gtin)
+    <td><div class=example>
+    ```json
+    ["urn:gtin:4712345060507"]
+    ```
+  <tr>
+    <td>UUID
+
+    Globally Unique Identifiers. See [[RFC9562]]
+    <td><div class=example>
+    ```json
+    ["urn:uuid:69585GB6-56T9-6958-E526-6FDGZJHU1326"]
+    ```
+  <tr>
+    <td>CAS Registry Number
+
+    Unique identification number assigned to every chemical substance described in the open scientific literature See [cas.org](https://www.cas.org/cas-data/cas-registry)
```

Comment:
CAS and InChI I don't believe are unique product identifiers, rather product classifications (i.e. classifications of chemical substances, but not specific to a given company for example)....So I think it could be confusing to include these examples within the product ID examples? Perhaps better to include these examples within a section about "productClassifications"?

Reply:
Let's check with the community, the CAS and InChi samples were explicitly included in the proposal for product id's. I can imagine you could use a CAS number for buying 100 liters of some chemical.